app/js/components/manageApps/AddAddon.jsx

```diff
@@ -16,12 +16,13 @@ export default class AddAddon extends Component {
     this.state = {
       disableClick: true,
       dropzoneRef: {},
-    }
+    };
     this.showMessage = this.showMessage.bind(this);
   }
 
   showMessage() {
-    if (this.props.files == null || this.props.files.length < 1) {
+    const { files } = this.props;
```

Comment:
Awesome refactoring here. The function looks cleaner.

Reply:
Thanks @LWanjiru